### DISCUSSION TITLE
Upcoming Meetings Legacy Widget - Title font size

### DISCUSSION THREAD

**Discussion by conrac01** [code4recovery/12-step-meeting-list#1120] [Q&A] [answered]
I've enabled 2 widgets side-by-side in column blocks using 2023 Theme WP 6.2 ("Upcoming Meetings" and "The Events Calendar")

Using Chrome Inspect I believe "Upcoming Meetings" uses h3 for widget title and I would like to use h6 - so it better matches the "Events Calendar" title.  Obviously, I'm not a php coder or I would have changed this.

How can I have my Upcoming Meetings title match the Events title?
Development site: https://help.aaalbanyny.org   (It's on the home page)

Thanks,
Ed Mac
AA Central Office - Albany, NY

**Comment by joshreisner** [comment, marked as answer]
Hi @conrac01 they are both `h2` tags, so you shouldn't need to change the code - the difference is that the events calendar is applying CSS to set the size of their `h2` to `16px` while ours is getting set by your theme. perhaps add some custom CSS to set ours to `16px` as well?

```
.tsml-widget-upcoming h2 {
  font-size: 16px;
}
```

**Reply by conrac01** [reply]
Thanks so much.  Perfect!  It looks great now.....
